src/Symfony/Installer/NewCommand.php

```diff
@@ -609,4 +609,15 @@ private function getExecutedCommand()
 
         return sprintf('%s new %s %s', $executedCommand, $this->projectName, $version);
     }
+
+    /**
+     * Checks whether the given directory is empty or not.
+     *
+     * @param  string  $dir the path of the directory to check
+     * @return boolean      True if the directory is empty. False otherwise.
+     */
+    private function isEmptyDirectory($dir)
+    {
+        return 0 === count(glob($dir.'/*', GLOB_NOSORT));
```

Comment:
Thanks for reporting this error. This PR has been updated to use `scandir()` instead of `glob()`